test/middlewares/schema_test.exs

```diff
@@ -253,6 +253,46 @@ defmodule Rajska.SchemaTest do
     end
   end
 
+  test "Skip the authorization process when query is '__typename'" do
```

Comment:
what do you think about renaming this to something like "Does not break and skips middleware check for subscriptions"? Since the fact that Absinthe does an instrospection on `__typename` is an implementation detail